docs/yaml.rst

```diff
@@ -25,59 +25,58 @@ Here is an example:
 .. code-block:: yaml
 
   bufr:
-    exports:
-      group_by_variable: longitude  # Optional
-      subsets:
-        - NC004001
-        - NC004002
-        - NC004003
-      variables:
-        timestamp:
-          datetime:
-            year: "*/YEAR"
-            month: "*/MNTH"
-            day: "*/DAYS"
-            hour: "*/HOUR"
-            minute: "*/MINU"
-            second: "*/SECO"  # default assumed zero if skipped or found as missing
-            hoursFromUtc: 0  # Optional
-
-        # Or, sometimes BUFR data use an offset time related to model analysis/cycle.
-        timestamp:
-          timeoffset:
-            timeOffset: "*/PRSLEVEL/DRFTINFO/HRDR"
-            transforms:
-              - scale: 3600
-            referenceTime: "2020-11-01T12:00:00Z"
-
-        satellite_id:
-          query: "*/SAID"
-          type: int64
-        longitude:
-          query: "*/CLON"
+    group_by_variable: longitude  # Optional
+    subsets:
+      - NC004001
+      - NC004002
+      - NC004003
+    variables:
+      timestamp:
+        datetime:
+          year: "*/YEAR"
+          month: "*/MNTH"
+          day: "*/DAYS"
+          hour: "*/HOUR"
+          minute: "*/MINU"
+          second: "*/SECO"  # default assumed zero if skipped or found as missing
+          hoursFromUtc: 0  # Optional
+
+      # Or, sometimes BUFR data use an offset time related to model analysis/cycle.
+      timestamp:
+        timeoffset:
+          timeOffset: "*/PRSLEVEL/DRFTINFO/HRDR"
           transforms:
-            - offset: -180
-        latitude:
-          query: "*/CLAT"
-        channels:
-          query: "[*/BRITCSTC/CHNM, */BRIT/CHNM]"
-        radiance:
-          query: "[*/BRITCSTC/TMBR, */BRIT/TMBR]"
-
-      splits:
-        satId:
-          category:
-            variable: satellite_id
-            map:
-              _3: sat_1  # can't use integers as keys
-              _5: sat_2
-              _8: sat_3
-
-      filters:
-        - bounding:
-            variable: longitude
-            upperBound: -68  # optional
-            lowerBound: -86.3  # optional
+            - scale: 3600
+          referenceTime: "2020-11-01T12:00:00Z"
+
+      satellite_id:
+        query: "*/SAID"
+        type: int64
+      longitude:
+        query: "*/CLON"
+        transforms:
+          - offset: -180
+      latitude:
+        query: "*/CLAT"
+      channels:
+        query: "[*/BRITCSTC/CHNM, */BRIT/CHNM]"
+      radiance:
+        query: "[*/BRITCSTC/TMBR, */BRIT/TMBR]"
+
+    splits:
+      satId:
+        category:
+          variable: satellite_id
+          map:
+            _3: sat_1  # can't use integers as keys
+            _5: sat_2
+            _8: sat_3
+
+    filters:
+      - bounding:
+          variable: longitude
+          upperBound: -68  # optional
+          lowerBound: -86.3  # optional
 
 The bufr section contains a section called **exports** which defines the data to read from the BUFR.
```

Comment:
Exports is removed now, right?

Reply:
Yes.  `export` was remove a while ago.

Reply:
The documentation got out sync with the YAML as Emily said. Will remove this blurb.